VMAccess/vmaccess.py

```diff
@@ -321,6 +332,41 @@ def _insert_rule_if_not_exists(rule_string):
     if cmd_result[0] == 0 and (rule_string not in cmd_result[1]):
         waagent.Run("iptables -I %s" %rule_string)
 
+def _fsck_check():
+     try:
+	retcode = waagent.Run("fsck -As -y")
+     if retcode >0:
+	raise Exception("Disk check was not successful")
+     else:
+	return retcode
+except Exception, e:
+        hutil.error("Failed to run disk check with error: %s" %(str(e), traceback.format_exc()))
+        hutil.do_exit(1, 'Check','error','0', 'Check failed.')
+
+
+def _fsck_repair():
+    #first unmount disks and loop devices lazy + forced
+    try:
+	cmd_result = waagent.Run("umount -dflr")
+	if cmd_result!=0:
+	#Fail fast
+		raise Exception("Failed to unmount disks")
+	else:
+	  # run repair
+	  retcode = waagent.Run("fsck -AR -y")
+	    if retcode >0:
+		raise Exception("Failed to repair disk")
+		else:
```

Comment:
it looks like this else should be at the same indent as the if on line 357?
Otherwise, this else will only be hit if retcode>0 (which doesn't make sense and will probably result in a crash) and only the next line will be the 'else block' (since it's not indented more)